isis/src/galileo/apps/gllssical/tsts/default/Makefile

```diff
@@ -3,7 +3,30 @@ APPNAME = gllssical
 include $(ISISROOT)/make/isismake.tsts
 
 commands:
+	# Test old spiceinited data
 	$(APPNAME) FROM=$(INPUT)/3439R.cub TO=$(OUTPUT)/3439R.cal.cub > /dev/null;
 	catlab FROM=$(OUTPUT)/3439R.cal.cub TO=$(OUTPUT)/3439R.cal.pvl > /dev/null;
 	$(APPNAME) FROM=$(INPUT)/1213r.cub TO=$(OUTPUT)/1213r.cal.cub > /dev/null;
 	catlab FROM=$(OUTPUT)/1213r.cal.cub TO=$(OUTPUT)/1213r.cal.pvl > /dev/null;
+
+	# Test newly re-spiceinited data
+	$(APPNAME) FROM=$(INPUT)/3439R.respiceinit.cub TO=$(OUTPUT)/3439R.respiceinit.cub > /dev/null;
```

Comment:
Was the original cube spiceinit'ed, but you couldn't get a camera off it?

Reply:
Yes, exactly. I figured we'd want to continue support gllssical working on older data without requiring that it be re-spiceinited.